### PR TITLE
docs: added one more domain to firewall section in troubleshoot

### PR DESCRIPTION
**Pull request by rupareddy5-21**
This domain will be hit when there's a new version of function core tools

### CODE REVIEW

File: docs/www/docs/contribute/99-troubleshooting.md [hunk 1]
```diff
@@ -110,21 +115,24 @@ For eg:
...
 ```
 PS C:\Users\USER> (Get-fileHash -Algorithm SHA256 .\.swa\deploy\VERSION\StaticSitesClient.exe).Hash.ToLower()
 ```
+
 2. On Linux:
+
 ```
 ➜ sha256sum ~/.swa/deploy/VERSION/StaticSitesClient | head -c 64
 ```
+
```
Comment (sulabh-msft) [inline docs/www/docs/contribute/99-troubleshooting.md:133]: Are these intentional changes or done as part of some formatting tool?

**Review by sulabh-msft — APPROVED** [review]

**Review by sgollapudi77 — APPROVED** [review]

File: docs/www/docs/contribute/99-troubleshooting.md (hunk 1 above)
Comment (rupareddy5-21) [reply docs/www/docs/contribute/99-troubleshooting.md:133]: not intentional Sulabh.

Comment (sulabh-msft) [reply docs/www/docs/contribute/99-troubleshooting.md:133]: Can we revert these changes and just keep one line change for the addition of the domain

Comment (rupareddy5-21) [reply docs/www/docs/contribute/99-troubleshooting.md:133]: even if I revert these changes are being added automatically when a commit is made.

Comment (sulabh-msft) [reply docs/www/docs/contribute/99-troubleshooting.md:133]: All right. Then lets go ahead with the changes then

File: docs/www/docs/contribute/99-troubleshooting.md [hunk 2]
```diff
@@ -110,21 +115,24 @@ For eg:
 }
 ```
 
-**IMPORTANT: Make sure the `StaticSitesClient.json#checksum` and `StaticSitesClient.json#metadata.files.[OS].sha` values match!**
-6. For Linux and macOS, run `chmod +x /home/USER/.swa/deploy/1.0.020761/StaticSitesClient`
-7. Run `swa deploy --verbose silly` and make sure `SWA_CLI_DEPLOY_BINARY` is set correctly. If everything was configured correctly, the deploy should work.
-
+**IMPORTANT: Make sure the `StaticSitesClient.json#checksum` and `StaticSitesClient.json#metadata.files.[OS].sha` values match!** 6. For Linux and macOS, run `chmod +x /home/USER/.swa/deploy/1.0.020761/StaticSitesClient` 7. Run `swa deploy --verbose silly` and make sure `SWA_CLI_DEPLOY_BINARY` is set correctly. If everything was configured correctly, the deploy should work.
```
Comment (sulabh-msft) [inline docs/www/docs/contribute/99-troubleshooting.md:118]: Just check this one why `6.For linux..` statement is moved up and even the 7th point is in a single line.

**Pull request merged**